src/main/java/com/karankumar/bookproject/backend/utils/PredefinedShelfUtils.java

```diff
@@ -36,13 +36,13 @@
 
 @Log
 public class PredefinedShelfUtils {
-    private final PredefinedShelfService predefinedShelfService;
+    private static PredefinedShelfService predefinedShelfService;
 
     public PredefinedShelfUtils(PredefinedShelfService predefinedShelfService) {
         this.predefinedShelfService = predefinedShelfService;
     }
```

Comment:
We don't want to instantiate this util class (see the issue description), so could you please remove the constructor?